apps/nextjs-website/src/components/Editorial.tsx

```diff
@@ -44,47 +43,8 @@ const makeEditorialProps = ({
   ...rest,
 });
 
-const Editorial = (props: EditorialSection) => {
-  const themeColor =
-    props.theme === 'dark' ? 'primary.contrastText' : 'text.primary';
-
-  return (
-    <section id={props.sectionID || undefined}>
-      <Stack
-        sx={{
-          section: {
-            ...(props.theme === 'dark' && {
-              backgroundColor: 'custom.backgroundColorDark',
-            }),
-          },
-          img: {
-            maxHeight: 490,
-            width: '100%',
-          },
-          '.MuiTypography-root': {
-            color: themeColor,
-            a: {
-              color: themeColor,
-              fontWeight: 'bold',
-            },
-          },
-          '.MuiTypography-h4': {
-            color: themeColor,
-          },
-          '.MuiTypography-body2': {
-            color: themeColor,
-          },
-          '.MuiTypography-overline': {
-            ...(props.theme === 'light' && {
-              color: 'text.secondary',
-            }),
-          },
-        }}
-      >
-        <EditorialEC {...makeEditorialProps(props)} />
-      </Stack>
-    </section>
-  );
-};
+const Editorial = (props: EditorialSection) => (
+  <EditorialEC {...makeEditorialProps(props)} />
```

Comment:
```suggestion
  <EditorialRC {...makeEditorialProps(props)} />
```
Consequence of suggestion above.